### PR TITLE
Scanne page

### PR DESCRIPTION
**Pull request by CzerwonyV**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -1,14 +1,10 @@
...
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <PageScan />
-      </header>
-    </div>
+    <>
+    </>
```
Comment (rocambille) [inline src/App.jsx:7]: this makes the App empty...

**Review by rocambille — CHANGES_REQUESTED** [review]
ScanePage: maybe ScanPage or ScannerPage, but the word scane doesn't exist

File: src/App.jsx (hunk 1 above)
Comment (CzerwonyV) [reply src/App.jsx:7]: yes, that's normal

Comment (rocambille) [reply src/App.jsx:7]: if you say so... still the name ScanePage is strange

Comment (CzerwonyV) [reply src/App.jsx:7]: Yes, that's true, but that's the name of my branch and for the moment I'm not made the ScanePage component.

**Pull request merged**